Header header
bool hasKeypointMeasurements
bool hasKeypointMeasurementUncertainties
bool hasKeypointOrientations
bool hasKeypointScores
bool hasKeypointScales
bool hasFloatDescriptors
bool hasBoolDescriptors
bool hasTrackIds
bool hasRawImage
bool hasChannel
int32[] keypointMeasurementsX
int32[] keypointMeasurementsY
int32 numKeypointMeasurements
float32[] keypointMeasurementUncertainties
float32[] keypointOrientations
float32[] keypointScores
float32[] keypointScales
std_msgs/Float32MultiArray descriptors
int32[] trackIds
sensor_msgs/Image rawImage
